### DISCUSSION TITLE
通过远程服务器自动更新课程表配置文件

### DISCUSSION THREAD

**Discussion by Jingbing1030** [Class-Widgets/Class-Widgets#290] [Feature]
1. 希望增加一个功能使软件可以获取远程服务器上的配置文件
2. 希望可以自定义时段（不只是划分为上午和下午）比如午自习划分较为细碎且跨越两个时间段，显示不够直观
3. 增加副课表，某些学校会按照第二课表来进行补课，而不是只是用一个课表（如第一周上语文，第二周上英语这样）
4. 允许自定义桌面显示课表所在的横纵坐标位置而不是只悬停在正上方中间

**Comment by QYlyj** [comment]
As for your third suggestion, the new function "mono-biweekly class schedule" in the new version 1.1.5 updated on August 28 meets your requirements. How to use: After setting the class schedule, you can open it in "Advanced Options".

**Comment by xuanxuan1231** [comment]
第一个可以试试rclone之类的，然后定时同步/l拉取

**Comment by Jingbing1030** [comment]
> 第一个可以试试rclone之类的，然后定时同步/l拉取

更需要的还是一个可以实时同步的课表（定时同步+启动时同步），这样就可以多班远程快速更改，而且也不需要每次更改课表都调整一体机的冰点配置。（比如像是希沃管家的冰点启动）

**Comment by Jingbing1030** [comment]
> As for your third suggestion, the new function "mono-biweekly class schedule" in the new version 1.1.5 updated on August 28 meets your requirements. How to use: After setting the class schedule, you can open it in "Advanced Options".

对于一个循环在两周以上的学校就不行了。
For schools with a curriculum cycle of more than two weeks, it is not possible.
